docs/nGraph_DG/nGraph_Python_API.md

```diff
@@ -0,0 +1,222 @@
+# Using nGraph's Python API
+
+nGraph is OpenVINO's graph manipulation library, used to represent neural network models in the form of a computational graph. Using nGraph's Python APIs, you can create, inspect and manipulate computational graphs. You can also use the API to set runtime properties of the graph, which will be used by OpenVINO during inference.
+
+## nGraph namespace module
+
+nGraph has a large set of mathematical operations, chosen to cover most ops used in popular neural network models. The `ngraph` module is the basic namespace used to expose commonly used API methods. You can inspect the `ngraph` module or call `help` on any method for usage information.
+
+```python
+import ngraph as ng
+dir(ng)
+help(ng.space_to_batch)
+```
+
+## Create a simple graph
+
+nGraph's Python API can be used to create computational models. Let's say you want to create a model which represents the following formula:
+
+    y = ReLU(x + b)
+
+You can use the following code to create parameters `x` and `b` and create the computational graph representing the formula:
+
+```python
+import numpy as np
+import ngraph as ng
+
+parameter_x = ng.parameter(shape=[2, 2], dtype=np.float32, name="x")
+parameter_b = ng.parameter(shape=[2, 2], dtype=np.float32, name="b")
+
+graph = ng.relu(parameter_x + parameter_b, name="y")
+```
+
+nGraph operations are organized into sets, which are released with new versions of OpenVINO. As operation signatures can change between operation sets, it's good practice to use operations from a specific operation set in your scripts. This will ensure that your code is compatible with future versions of OpenVINO.
+
+The example above can be rewritten like this to use only operations from opset 4:
+
+```python
+import ngraph.opset4 as ops
+
+parameter_x = ops.parameter(shape=[2, 2], dtype=np.float32, name="x")
+parameter_b = ops.parameter(shape=[2, 2], dtype=np.float32, name="b")
+
+graph = ops.relu(parameter_x + parameter_b, name="y")
+```
+
+
+## Create an nGraph function from a graph
+
+You can combine a graph together with it's input parameters into an nGraph function object. An nGraph function can be passed to other parts of OpenVINO to perform analysis and inference.
+
+```python
+>>> function = ng.Function(graph, [parameter_x, parameter_b], "TestFunction")
+<Function: 'TestFunction' ({2,2})>
+```
+
+## Run inference
+
+In order to run inference on an nGraph function, you'll need to convert it to an Inference Engine network
```

Comment:
To run inference on an nGraph function, convert it to an Inference Engine network